src/render/mask.rs

```diff
@@ -34,7 +34,7 @@ pub fn create(
     let mut rc = ResourceContainer::new();
 
     let mut content = Content::new();
-    content.save_state();
+    content.save_state_checked()?;
```

Comment:
I presume you've done a Ctrl + F to make sure you caught all instances of `save_state`? 😄

Reply:
I did